runner/runner.go

```diff
@@ -274,7 +275,7 @@ func errorResult(message string) *gauge_messages.ProtoExecutionResult {
 
 // Looks for a runner configuration inside the runner directory
 // finds the runner configuration matching to the manifest and executes the commands for the current OS
-func StartRunner(manifest *manifest.Manifest, port string, reporter reporter.Reporter, killChannel chan bool, debug bool) (*LanguageRunner, error) {
+func StartRunner(manifest *manifest.Manifest, port string, reporter io.Writer, killChannel chan bool, debug bool) (*LanguageRunner, error) {
```

Comment:
rename `reporter` to something meaningful, since it is no longer `reporter.Reporter`